docs/reference/garbage-collection.md

```diff
@@ -191,20 +191,40 @@ spark-submit --class io.treeverse.clients.GarbageCollector \
 
 The list of expired objects is written in Parquet format in the storage
 namespace of the bucket under
-`_lakefs/retention/gc/addresses/run_id=RUN_ID`, where RUN_ID identifies the
+`_lakefs/retention/gc/addresses/mark_id=MARK_ID`, where MARK_ID identifies the
 run.
 
-### First run
+### GC job options
 
-To create a list of objects to delete you can
+By default, GC first creates a list of expired objects according to your retention rules and then hard-deletes those objects. 
+However, you can use GC options to break the GC job down into two stages: 
+1. Mark stage: GC will mark the expired objects to hard-delete, **without** deleting them. 
+2. Sweep stage: GC will hard-delete objects marked by a previous mark-only GC run. 
 
-* Add `-c spark.hadoop.lakefs.debug.gc.no_delete=true` to disable object
-  deletion.
+By breaking GC into these stages, you can pause and create a backup of the objects that GC is about to sweep and later 
+restore them. You can use [GC backup and restore](#gc-backup-and-restore) utility for that.   
 
-Nothing will be deleted.  The list of expired objects will still be created
-and may be examined.
+#### Mark only mode 
 
-### Performance
+To make GC run the mark stage only, add the following properties to your spark-submit command:
+```properties
+spark.hadoop.lakefs.gc.do_sweep=false
```

Comment:
Definitely need to note the minimal version for mark-only. Otherwise I can set it on an older version and be unpleasantly surprised.

Reply:
great call, added